build.cmd

```diff
@@ -27,7 +27,7 @@ src\.nuget\NuGet.exe update -self
 src\.nuget\NuGet.exe install FAKE -ConfigFile src\.nuget\Nuget.Config -OutputDirectory src\packages -ExcludeVersion -Version 4.16.1
 
 src\.nuget\NuGet.exe install xunit.runner.console -ConfigFile src\.nuget\Nuget.Config -OutputDirectory src\packages\FAKE -ExcludeVersion -Version 2.0.0
-src\.nuget\NuGet.exe install NBench.Runner -OutputDirectory src\packages -ExcludeVersion
+src\.nuget\NuGet.exe install NBench.Runner -OutputDirectory src\packages -ExcludeVersion -Version 0.1.5
```

Comment:
Locked on 0.1.5 - will need to be upgraded later if we update the version of NBench